src/components/NewGalaxyModal.js

```diff
@@ -21,16 +21,15 @@ const styles = {
   headerText: { fontSize: 16, fontWeight: 600 }
 }
 
-// TODO: Once we have more appTypes, abstract text/component to support other apps
-export const NewAppModal = _.flow(
-  Utils.withDisplayName('NewAppModal'),
+export const NewGalaxyModal = _.flow(
+  Utils.withDisplayName('NewGalaxyModal'),
   withModalDrawer({ width: 675 })
 )(({ onDismiss, onSuccess, apps, workspace: { workspace: { namespace, bucketName, name: workspaceName } } }) => {
   const [viewMode, setViewMode] = useState(undefined)
   const [loading, setLoading] = useState(false)
 
   const app = currentApp(apps)
```

Comment:
I wasn't sure if this should change. Given that it is still currentApp, but this does keep the abstraction of app here. May be okay cause its used in the scope of `NewGalaxyModal`?